### PR TITLE
Update jobber.py

### LINKED ISSUE #51: 📚 Documentation: Write inline documentation for fedrec/multiprocessing/jobber.py
# Where?

HERE-
- https://github.com/NimbleEdge/EnvisEdge/blob/00f554973085c54da98caf37721bec27ba07380d/fedrec/multiprocessing/jobber.py#L62
- https://github.com/NimbleEdge/EnvisEdge/blob/00f554973085c54da98caf37721bec27ba07380d/fedrec/multiprocessing/jobber.py#L79
- https://github.com/NimbleEdge/EnvisEdge/blob/00f554973085c54da98caf37721bec27ba07380d/fedrec/multiprocessing/jobber.py#L86

# Who?

Anyone who wants to get know the code base of NimbleEdge as well as any file in the project.

# What?

Please follow these guidelines: [LInk](https://numpydoc.readthedocs.io/en/latest/format.html)
We need to go "line-by-line" and create inline documentation for this file. Please try to use proper punctuation and English grammar to the best of your ability.

# Additional Content

We need to go "line-by-line" and create inline documentation for this file. Please try to use proper punctuation and English grammar to the best of your ability.

We are using the pep8 format for documentation. The following links are concise and would be very useful:

pep8 [guidelines](https://www.python.org/dev/peps/pep-0008/)
numpy style [guide](https://numpydoc.readthedocs.io/en/latest/format.html)

### PR DESCRIPTION
**Pull request by abhiwalia15**
Created inline documentation for this file. I have followed all the instructions.

## Description
Fixes #51 
- Followed the numpystyle documentation style to write inline documentation for the jobbers.py file.
- Text is grammatically correct and has been checked using writing software.

## Relevant Issue
-Partially fixes #182 

## Affected Dependencies
This work is independent of any other files or folders in this project. 

## Checklist
- [+] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/EnvisEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
- [+] I have commented on my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)
- [+] I have labelled this PR with the relevant [Type labels](https://github.com/NimbleEdge/EnvisEdge/labels)
- [+] My changes are covered by tests

### CODE REVIEW

File: fedrec/multiprocessing/jobber.py [hunk 1]
```diff
@@ -1,3 +1,8 @@
+"""Jobber python file for scheduling job requrests to run
```
Comment (ramesht007) [inline fedrec/multiprocessing/jobber.py:1]: It should be python module

File: fedrec/multiprocessing/jobber.py [hunk 2]
```diff
@@ -79,6 +92,10 @@ def execute(self, message: JobSubmitMessage):
     def publish(self, job_result: JobResponseMessage) -> None:
         """
         Publishes the result after executing the job request
```
Comment (ramesht007) [inline fedrec/multiprocessing/jobber.py:95]: this can be ```publishes the result on kafka after executing the job request```

**Review by ramesht007 — CHANGES_REQUESTED** [review]

**Comment by ramesht007** [comment]
Please resolve the conflict that is coming.

**Comment by ramesht007** [comment]
and please write a one liner comment for this method.
https://github.com/NimbleEdge/EnvisEdge/blob/00f554973085c54da98caf37721bec27ba07380d/fedrec/multiprocessing/jobber.py#L86

File: fedrec/multiprocessing/jobber.py (hunk 1 above)
Comment (abhiwalia15) [reply fedrec/multiprocessing/jobber.py]: Done!

File: fedrec/multiprocessing/jobber.py (hunk 2 above)
Comment (abhiwalia15) [reply fedrec/multiprocessing/jobber.py]: Done!

**Comment by abhiwalia15** [comment]
> Please resolve the conflict that is coming.

Conflicts Resolved.

**Comment by abhiwalia15** [comment]
> and please write a one liner comment for this method.
> 
> https://github.com/NimbleEdge/EnvisEdge/blob/00f554973085c54da98caf37721bec27ba07380d/fedrec/multiprocessing/jobber.py#L86

Made requested Changes.

File: fedrec/multiprocessing/jobber.py [hunk 3]
```diff
@@ -101,4 +101,8 @@ def publish(self, job_result: JobResponseMessage) -> None:
         self.comm_manager.send_message(job_result)
 
     def stop(self) -> None:
+        '''
+        This function is called after the end of a job request
+        to perform cleanup, logging, etc.
```
Comment (ramesht007) [inline fedrec/multiprocessing/jobber.py:109]: Good work @abhiwalia15 
Just remove etc from line 106.
and we are good to go

Comment (abhiwalia15) [reply fedrec/multiprocessing/jobber.py]: Made the required changes. Thanks

**Review by ramesht007 — APPROVED** [review]
LGTM!!

**Comment by abhiwalia15** [comment]
> LGTM!!

Thanks!

**Pull request merged**